From 50301b5f0e32d53d2cdada1f61b9a332d292c816 Mon Sep 17 00:00:00 2001
From: David Wu <david.wu@rock-chips.com>
Date: Mon, 27 Aug 2018 21:43:09 +0800
Subject: [PATCH] dt-bindings: net: rockchip-dwmac: Add gmac support for rk1808

This patch adds devicetree support to gmac of rk1808 with proper
devicetree compatible strings.

Change-Id: Id35523a10f987cbb9b9c33ad32ab23cb3c6d4e2b
Signed-off-by: David Wu <david.wu@rock-chips.com>
---
 Documentation/devicetree/bindings/net/rockchip-dwmac.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/net/rockchip-dwmac.txt b/Documentation/devicetree/bindings/net/rockchip-dwmac.txt
index 433782360e3d..43a74c66677d 100644
--- a/Documentation/devicetree/bindings/net/rockchip-dwmac.txt
+++ b/Documentation/devicetree/bindings/net/rockchip-dwmac.txt
@@ -5,6 +5,7 @@ The device node has following properties.
 Required properties:
  - compatible: should be "rockchip,<name>-gmac" or "rockchip,<name>-mac"
    "rockchip,px30-gmac": found on PX30 SoCs
+   "rockchip,rk1808-gmac": found on RK1808 SoCs
    "rockchip,rk3128-gmac": found on RK312x SoCs
    "rockchip,rk3228-gmac": found on RK322x SoCs
    "rockchip,rk3288-gmac": found on RK3288 SoCs
-- 
2.35.3

